less/middle-content.less

```diff
@@ -0,0 +1,320 @@
+.home {  
+    margin-left: 7%;
+    margin-right: 7%;
+
+    @media @phone {
+      margin-top: 10%;
+    }
+  
+    .middle-content {    
+      height: 100vh;
+      width: 100%;
+      display: flex;
+      justify-content: space-around;
+      flex-flow: row wrap;
+      padding-bottom: 80%;
+
+      @media @phone {        
+        padding-bottom: 320%;
```

Comment:
If you have to do something like this, that always means there is an issue somewhere up the line that is the root of all your problems.